Cargo.toml

```diff
@@ -60,6 +60,7 @@ git = "https://github.com/gtk-rs/sys"
 [dependencies]
 libc = "0.2"
 bitflags = "1.0"
+thiserror = "1.0.10"
```

Comment:
So we're adding new proc-macro dependencies? :'(

Reply:
@GuillaumeGomez Sorry, I totally forgot about this comment.
You don't want this dependence?
I can only remove `fn description` and change error texts in `Display`

Reply:
It's always advantages versus downsides. For me, it doesn't bring enough to justify it but @sdroege might disagree on this.

Reply:
It removes some boring boilerplate and does not really add a lot dependency-wise. We depend on all the dependencies apart from `thiserror` already IIRC (in glib/gio).

In case of switching over `gstreamer-rs` to `thiserror`, it also solved an annoying bug that nobody noticed in the manual implementation (infinite recursion in the `Display` impl, indirectly calling into itself).

Reply:
Then it's all good.